### ISSUE TITLE
Targetting doesn't work for me in razor [BUG]

### ISSUE THREAD

**Issue by Plaje** [markdwags/razor#216]
Using the old UO client on a local servuo server and the latest version of razor. None of the targetting features work at all for me and I don't know why.

-For example using "Target closest non-friendly" near a bunch of skeletons ends up targetting a llama (verified by using I think set last target where it prints the name), which I haven't seen any llamas in several hours, if any at all in the session. 
-Using "target by type" near the same group of yew skeletons results in the game saying there are no targets of that type within range. The mob number is 50 in the macro window after setting the target by type on a skeleton, if that is helpful.
-Last target works sometimes but most of the time doesn't (results in "target out of range" even though the previously-set last target is standing next to me). I haven't really figured out why, I wasn't doing anything different or special when it did work as far as I can tell.
-Target self usually does work. I'm not sure why this is different but I'm glad I have at least that functionality.
-Of course the result is the game says the target is out of range for most of these actions.

I'm not doing anything special here. Just doing stuff like use skill provocation -> last target, or taming -> last target, etc. Turning smart target options on and off does not help.

Maybe there is something different about my setup since I don't see anyone else complaining about it, but I'm using the default settings of servuo and razor aside from the IP and port being changed. Maybe servuo is different from what production servers are using? But I don't know why they would be, so I don't know what is causing the issue. It seems to me like it should work. The macro in the picture was used with Last Target set to the cat next to me.
![New Bitmap Image](https://github.com/markdwags/Razor/assets/52856622/1f5319c0-6a49-4fed-b53c-5ec1884f0342)

**Comment by markdwags** [comment]
Based on that screenshot, you're using the old UO client, not ClassicUO.

**Comment by Plaje** [comment]
Sorry you are correct, it is the old UO client. Updated the post. I'll try running ClassicUO to see if that changes anything. I will note I am using the x86 version of razor as instructed on the download page

**Comment by Plaje** [comment]
OK yeah, it does work on ClassicUO, apparently the x86 version doesn't target well with the old client

**Comment by Plaje** [comment]
Minor note/update: didn't think about this before but the automatic door opening for x86 version didn't work on the old uo client either, I just noticed that is working with x64 on the classicUO client. Well, I'm aware you guys said support is dropping off for x86 on the download page but just letting you know. I'm happy using the classicUO client for now. Sorry I got the terminology mixed up there earlier.
